conda/core/solve.py

```diff
@@ -1029,6 +1029,10 @@ def _notify_conda_outdated(self, link_precs):
 
                     $ conda update -n base -c %s conda
 
+                Or to minimize the number of packages updated during conda update use
+
+                     conda install conda=version
+
                 """) % (CONDA_VERSION, latest_version, channel_name), file=sys.stderr)
```

Comment:
If we are touching this part of the code, let's go ahead and use f-strings here. I'll let you figure out how to do that 😄  (feel free to ask if you need help though).

Also, I'm pretty sure that:

```
conda install conda=version
```

Won't actually install the latest version, right? This was the error I got when trying to run it:

```
PackagesNotFoundError: The following packages are not available from current channels:

  - conda=version
```

Reply:
Hah! https://github.com/conda/conda/pull/11796/commits/9e61f919db778bc8a5d13a3b81cab25d6023a386